connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -79,6 +84,15 @@
 @RunWith(PowerMockRunner.class)
 public class WorkerSourceTaskTest extends ThreadedTest {
     private static final String TOPIC = "topic";
+    private static final Headers EMPTY_HEADERS = new RecordHeaders();
+    private static final Headers HEADERS = new RecordHeaders();
+    static {
+        HEADERS.add("header_key", "header_value".getBytes());
+    }
+    private static final org.apache.kafka.connect.header.Headers CONNECT_HEADERS = new ConnectHeaders();
+    static {
+        CONNECT_HEADERS.add("header_key", new SchemaAndValue(Schema.STRING_SCHEMA, "header_value"));
+    }
```

Comment:
Similar comment as above w/r/t these are strictly speaking not immutable.

Reply:
Refactored to use `emptyHeaders()` + specific headers for the test